pkg/server/debug/server.go

```diff
@@ -133,6 +134,7 @@ func setupProcessWideRoutes(
 		_ = dump.HTML(w)
 	}))
 
+	mux.HandleFunc("/debug/pprof/fgprof", authzFunc(fgprof.Handler().ServeHTTP))
```

Comment:
`fgprof` has very high overhead to the extent it's not suitable to be used in production. There should be some comment to that effect to warn the user.

Reply:
Added something.